arviz/stats/density_utils.py

```diff
@@ -71,7 +71,7 @@ def _bw_isj(x, grid_counts=None, x_std=None, x_range=None):
     a_sq = a_k[range(1, grid_len)] ** 2
 
     t = _root(_fixed_point, x_len, args=(x_len, k_sq, a_sq), x=x)
-    h = t**0.5 * x_range
+    h = t ** 0.5 * x_range
```

Comment:
emm, not sure if I should revert all changes here, `black` automatically reformat this locally for me.

Reply:
Revert, apparently black is not happy. Probably black version in CI is different? 🤔